documentation/ame-guide/modules/ROOT/pages/modeling/working-with-multiple-windows.adoc

```diff
@@ -1,24 +1,25 @@
 [[new-window]]
-= Working with Multiple Windows
+= Working with multiple windows
```

Comment:
I have the feeling the formatting for UI text is here not consistent, maybe double-check

Reply:
What do you mean? This?

![grafik](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/101344507/0a0396a0-51c8-4b4d-98c3-17c0f4044f20)

And yes, in the navigation bar, it should rather be "Multiple windows" (lower case, sentence style). Consider creating an issue to review headings and navigation for sentence style.